lib/ui/io.rb

```diff
@@ -0,0 +1,13 @@
+class IO
+  def initialize(input, output)
+    @input, @output = input, output
+  end
+
+  def display_user_message(message)
+    @output.print message
+  end
+
+  def get_user_input
```

Comment:
Both of these method names have `user` in them.  What do you think about that?

Reply:
I wasn't a fan.  I would rather just call them display_message and get_input.  Will update and see what you all think.